apps/web-giddh/src/app/mobile-home/mobile-home.component.html

```diff
@@ -0,0 +1,82 @@
+<div class="mobile-home-search">
+        <div class="company-name-alice btn-group" dropdown>
+            <p dropdownToggle>{{companyInitials}}</p>
+            <ul id="dropdown-basic" *dropdownMenu class="dropdown-menu"
+                role="menu" aria-labelledby="button-basic">
+                <li role="menuitem"><a class="dropdown-item" href="javascript:void(0)" (click)="logout()">Logout</a></li>
+            </ul>
+        </div>
+        <div class="account-search">
+            <div class="input-group" (click)="focusInSearchBox()" [ngClass]="{'group-list': listOfSelectedGroups && listOfSelectedGroups.length > 0}">
+                <div class="wrap-group-account-input">
+                    <span class="item-search-wrapper" *ngFor="let item of listOfSelectedGroups; let idx = index">
+                        <span>
+                            <span class="item-search-name">{{item.name}}</span>
+                            <ng-container>
+                                    <span>&nbsp;❯&nbsp;</span>
+                            </ng-container>
+                        </span>
+                    </span>
+                    <input type="text" #searchElement class="form-control" [(ngModel)]="searchString" placeholder="Search" aria-describedby="" (keyup)="initSearch($event, $event.target.value)" (keydown)="handleKeydown($event)">
```

Comment:
aria-describedby="" blank if not usable pls remove

Reply:
Resolved